adColony/ios/src/com/submarine/adcolony/AdColonyIosNetwork.java

```diff
@@ -0,0 +1,41 @@
+package com.submarine.adcolony;
+
+/**
+ * Created by mariam on 3/9/15.
+ */
```

Comment:
it will be nice if you rename this to IOSAdColonyNetwork, use IOS all in capital letters to keep convension